halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/NestableCommand.java

```diff
@@ -37,7 +37,6 @@
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.Setter;
-import org.apache.commons.lang.NotImplementedException;
```

Comment:
Swapped this for `UnsupportedOperationException` since jackson seems to enter an infinitely recursive loop when trying to serialize `NotImplementedException`.